src/commands/shortcut.js

```diff
@@ -1,34 +1,29 @@
-import _ from 'lodash';
-import inquirer from 'inquirer';
-import {
-  loadExtensionJsonAsync,
-  saveExtensionJsonAsync,
-} from '../extension/data';
+import * as shortcut from '../extension/shortcut';
+import { loadExtensionJsonAsync, saveExtensionJsonAsync } from '../extension/data';
 import msg from '../user_messages';
 
-export async function createShortcut(shortcutName, screenName = null) {
+export async function createShortcut(shortcutName) {
```

Comment:
Already refactoring authorization in another branch.
Also, extension/cache with it's setValue and getValue is implemented to be used as state manager.